server/controllers/courseCtrl.js

```diff
@@ -620,6 +630,7 @@ async function getIndex(req, res, next) {
         lang === 'sv'
           ? 'KTH kursinformation för ' + courseCode.toUpperCase()
           : 'KTH course information ' + courseCode.toUpperCase(),
+      cookies: klaroConsentCookie,
```

Comment:
Because it's specifically analytics consent, it is better to rename `cookies` to `klaroAnalyticsConsentCookie` to keep it consistent. (In case in future we will need other type of consent cookie.)